Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -107,14 +112,250 @@ public void TearDown()
                 }
             }
             
-            // Delete temp files
-            File.Delete(_inputFilePath);
-            File.Delete(_outputFilePath);
+            // Delete temp files if necessary
+            if (_filesToDelete == null)
+                return;
+            
+            foreach (var file in _filesToDelete)
+            {
+                File.Delete(file);
+            }
+        }
+
+        [Test]
+        public void TestPutFileAsteriskWildcard()
+        {
+            // Prepare the data files to be copied
+            var absolutePath = $"{Path.GetTempPath()}{Guid.NewGuid()}";
```

Comment:
should we added to variable name info that it is prefix? because it seems to be absolute path to some file but there won't be a file named like that one

Reply:
Good idea, added `Prefix` to the variable name